README.md

```diff
@@ -1,15 +1,17 @@
-# signac - simple data management
+# <img src="doc/images/logo.png" width="100" height="100"> Signac - Simple Data Management

```

Comment:
@csadorf Did you mean to capitalize the name of the package? It's consistently lowercase everywhere else.

Reply:
No, it should not be capitalized. My mistake.